oak_crypto/src/lib.rs

```diff
@@ -35,6 +36,7 @@ pub mod proto {
 pub mod encryption_key;
 pub mod encryptor;
 pub mod hpke;
+pub mod noise_nk;
```

Comment:
nit: I think the whole module could just be called `noise` (and it will have both NK and KK inside it)